configuration/messageproperties/messages_zl_en.properties

```diff
@@ -164,3 +164,5 @@ ui.i18n.Location.name.387956c5-c991-4a2d-a03e-c53c11200ef7=UMI
 ui.i18n.Location.name.88f74254-60fb-4d31-bfde-3e75db5f5f4c=UMI
 ui.i18n.Location.name.8b2a0bac-9644-11eb-b8a1-0242ac110002=UMI
 
+zl.consultNote.delivery.label=Delivery
```

Comment:
Use pihcore.anc.delivery.title.  You don't need to add anything new, @louidorjp 

- messages_fr.properties:  pihcore.anc.delivery.title=Accouchement
- messages_en.properties: pihcore.anc.delivery.title=Delivery